platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/BaseBench.java

```diff
@@ -126,6 +139,18 @@ public void setup() throws IOException {
             registry.registerConstructables("com.swirlds.merkledb");
             registry.registerConstructables("com.swirlds.benchmark");
             registry.registerConstructables("com.swirlds.common.crypto");
+            registry.registerConstructables("com.swirlds.common");
+            registry.registerConstructable(new ClassConstructorPair(QueryResponse.class, QueryResponse::new));
+            registry.registerConstructable(
+                    new ClassConstructorPair(BenchmarkMerkleInternal.class, BenchmarkMerkleInternal::new));
+            registry.registerConstructable(
+                    new ClassConstructorPair(BenchmarkMerkleLeaf.class, BenchmarkMerkleLeaf::new));
+            registry.registerConstructable(new ClassConstructorPair(VirtualLeafRecord.class, VirtualLeafRecord::new));
+            registry.registerConstructable(new ClassConstructorPair(VirtualMap.class, VirtualMap::new));
+            registry.registerConstructable(new ClassConstructorPair(VirtualMapState.class, VirtualMapState::new));
+            registry.registerConstructable(new ClassConstructorPair(VirtualRootNode.class, VirtualRootNode::new));
+            registry.registerConstructable(new ClassConstructorPair(BenchmarkKey.class, BenchmarkKey::new));
+            registry.registerConstructable(new ClassConstructorPair(BenchmarkValue.class, BenchmarkValue::new));
```

Comment:
I wonder if `registerConstructable()` with a package name can be used instead of these individual constructor pairs? For example, `VirtualLeafRecord` is in `com.switlds.virtualmap.datasource`, so would `registerConstructables("com.swirlds.virtualmap.datasource")` work by any chance?

Reply:
They sure would. However, we're losing compile-time checks (javac wouldn't bother checking if the string literals refer to actual Java packages), and we're potentially getting runtime errors if someone adds classes to these packages that are incompatible with the constructable registry for whatever reasons. But okay.